scalableminds/webknossos
# Add cross origin api

Can be used via postMessage by a website embedding wK in an iFrame. Currently the only use case is the contactome viewer.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a volume tracing and open the dev console.
  - Execute `window.postMessage({type: "setMapping", args: [{1308340: 1308340, 1229920: 1229920}, {hideUnmappedIds: true}]}, "*")`, adjust the ids accordingly. You should see the result :)
  - You should not be able to execute other functions.

------
